.github/workflows/postman-collection-runner.yml

```diff
@@ -0,0 +1,36 @@
+name: Postman Collection Api test 
+
+on:
+  pull_request:
+    branches:
+      - main
+  schedule:
+      - cron: "15 0 * * *"
+
+env:
+  TZ: 'Asia/Kolkata' 
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  api_tests:
+    name: Run Postman Collection on integ env
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3.1.0
+      
+      - name: Install Newman
+        run: npm install -g newman
+      
+      - name: Install portman
+        run: npm install -g @apideck/portman
+      
```

Comment:
Can we use some sort of caching for these npm installs...

Reply:
Not sure let me check

Reply:
mostly saving time I guess from npm downloads (but i assume github would have a proxy cache for npm registry anyway...)

Can ignore this..

Reply:
newman should already be part of the base ubuntu image

https://github.com/actions/runner-images/blob/main/images/linux/Ubuntu2204-Readme.md#tools